src/PendingObjects/TestCall.php

```diff
@@ -84,6 +84,20 @@ public function with($data): TestCall
         return $this;
     }
 
+    public function dependsOn(string ...$tests): TestCall
```

Comment:
Lets remove this, and just have the `depends` for now.

Reply:
agree